src/dotnet/ToolPackage/ToolPackageFactory.cs

```diff
@@ -11,12 +12,13 @@ namespace Microsoft.DotNet.ToolPackage
     internal static class ToolPackageFactory
     {
         public static (IToolPackageStore, IToolPackageInstaller) CreateToolPackageStoreAndInstaller(
-            DirectoryPath? nonGlobalLocation = null)
+            DirectoryPath? nonGlobalLocation = null,
+            IEnumerable<string> alwaysForwardArguments = null)
```

Comment:
Can we just call this something like `restoreArguments` or `additionalRestoreArguments`?  "Forward" or "forwarded" in this context no longer has meaning, in my opinion.

Reply:
fixed, rename to additionalRestoreArguments